LayoutTests/platform/glib/TestExpectations

```diff
@@ -3470,6 +3470,7 @@ webkit.org/b/261024 svg/text/small-fonts-in-html5.html [ Failure ImageOnlyFailur
 webkit.org/b/261024 webrtc/video-maxFramerate.html [ Pass Timeout ]
 webkit.org/b/261024 webrtc/video-replace-muted-track.html [ Pass Timeout ]
 webkit.org/b/261024 webrtc/video-unmute.html [ Pass Timeout ]
+webkit.org/b/261024 webrtc/video-disabled-black.html [ Skip ]
```

Comment:
We usually prefer to indicate the possible outcomes of the test (Crash Pass) rather than skipping it.

If a test is skipped then the bots will not run it, so we will not get any further notification if/when the test starts passing back or if it has a different result (like Timeout).

Reply:
Done :) Thanks for the note. Now this just needs proper labels to merge, e.g. `unsafe-merge-queue`, would appreciate that if you'd add those labels @clopez (and also remove `merging-blocked` label if applicable). Thanks!